nebula/ui/components/MZSubscriptionOption.qml

```diff
@@ -25,21 +25,21 @@ RadioDelegate {
 
     background: Rectangle {
         id: bg
-        color: MZTheme.colors.white
+        color: MZTheme.colors.bgColorStronger
         radius: MZTheme.theme.cornerRadius
 
         MZRectangularGlow {
             anchors.fill: bg
             glowRadius: radioDelegate.checked ? 8 : 1
             spread: radioDelegate.checked ? 0.1 : 0
-            color: MZTheme.colors.grey30
```

Comment:
I think this is correct, but it's just different from most other drop shadows, which were `grey60`

Reply:
This was an inconsistency I streamlined when doing the PR. I tested it - it's a nearly imperceptible difference, and I took the liberty to slim down the colors used by reusing `grey60` here. Let me know if you don't like it, or want to see the side-by-side screenshots.